sdk/daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/package.scala

```diff
@@ -42,12 +42,12 @@ package object archive {
   // This constant is introduced and used
   // to make serialization of nested data
   // possible otherwise complex models failed to deserialize.
-  private val PROTOBUF_RECURSION_LIMIT: Int = 1000
+  private val EXTENDED_PROTOBUF_RECURSION_LIMIT: Int = 1000
```

Comment:
Appears to be a rename on 3.4 and main branches - hence copied over for consistency